### PR TITLE
Bugfix/dashboard responsive

### PR DESCRIPTION
**Pull request by martinbobbio**
- Fix responsive

### CODE REVIEW

File: src/app/shared/dashboard-list-item/dashboard-list-item.component.html [hunk 1]
```diff
@@ -77,14 +76,13 @@
...
-      <div class="list__item__column__title col-md-12 col-6 d-sm-none">
+      <div class="list__item__column__title col-md-12 col-6 d-lg-none">
         REPAYMENT PROGRESS
       </div>
 
       <div class="col-6">
-        <!-- TODO: check tooltip logic -->
-        <!-- [matTooltip]="'Accrued interest: ' + paymentProgress + ' ' + loan?.currency" -->
         <p
+          [matTooltip]="'Accrued interest: ' + paymentProgress"
```
Comment (nicolaswww) [inline src/app/shared/dashboard-list-item/dashboard-list-item.component.html:85]: @martinbobbio check this copy

File: src/app/shared/dashboard-list-item/dashboard-list-item.component.html [hunk 2]
```diff
@@ -39,13 +39,12 @@
       </div>
     </div>
 
-    <!-- TODO: check tooltip logic -->
-    <!-- [matTooltip]="'Effective rate: ' + anualRate" -->
     <div
       class="list__item__column"
+      [matTooltip]="'Effective rate: ' + anualRate"
```
Comment (nicolaswww) [inline src/app/shared/dashboard-list-item/dashboard-list-item.component.html:44]: @martinbobbio check this copy

File: src/app/shared/dashboard-list-item/dashboard-list-item.component.html [hunk 3]
```diff
@@ -107,13 +105,7 @@
...
             Pay
           </button>
-          <app-redeem-button
-            (startRedeem)="startRedeem()"
-            [loan]="loan"
-            *ngIf="canRedeem"
-            name="Withdraw Collateral"
-            className="dashboard"
-          >
+          <app-redeem-button [loan]="loan" *ngIf="canRedeem" name="Withdraw Collateral">
```
Comment (nicolaswww) [inline src/app/shared/dashboard-list-item/dashboard-list-item.component.html:108]: @martinbobbio the flag canRedeem changes when startRedeem() is called, please add this

**Review by nicolaswww — CHANGES_REQUESTED** [review]

File: src/app/shared/dashboard-list-item/dashboard-list-item.component.html (hunk 1 above)
Comment (martinbobbio) [reply src/app/shared/dashboard-list-item/dashboard-list-item.component.html:85]: Removed tooltip and add ToDo

File: src/app/shared/dashboard-list-item/dashboard-list-item.component.html (hunk 2 above)
Comment (martinbobbio) [reply src/app/shared/dashboard-list-item/dashboard-list-item.component.html:44]: Removed tooltip and add ToDo

**Review by nicolaswww — APPROVED** [review]

**Pull request merged**